### PR TITLE
Fix extended_stay_america

### PR DESCRIPTION
**Pull request by Cj-Malone**

### CODE REVIEW

File: locations/spiders/extended_stay_america.py [hunk 1]
```diff
@@ -1,52 +1,16 @@
-# -*- coding: utf-8 -*-
-import json
-import re
+from scrapy.spiders import SitemapSpider
 
-import scrapy
+from locations.structured_data_spider import StructuredDataSpider
+from locations.user_agents import BROSWER_DEFAULT
```
Comment (jleedev) [inline locations/spiders/extended_stay_america.py:4]: > BROSWER

Comment (Cj-Malone) [reply locations/spiders/extended_stay_america.py:4]: Well... fuck

Comment (Cj-Malone) [reply locations/spiders/extended_stay_america.py]: I've fixed it here, and I'll put it on my TODO list to fix it in the other spiders

**Comment by Cj-Malone** [comment]
Looks like the GitHub CI ip is now being blocked by them, we can hope that it's better on the AWS server for the weekly, but it's probably one that needs to be split into several proxies.

**Pull request merged**